cutevariant/gui/mainwindow.py

```diff
@@ -140,8 +140,9 @@ def open(self, filepath):
         if not os.path.exists(filepath):
             return
 
-        # Show the project name in title
+        # Show the project name in title and in status bar
         self.setWindowTitle(f"Cutevariant - %s" % os.path.basename(filepath))
+        self.status_bar.showMessage(self.tr(f"{filepath} opened"))
```

Comment:
I m not sure this will works with translation . 
I guess it should be :  

        showMessage(f"{filepath}" + self.tr("opened"))

Reply:
Yes i m testing this in a minute